map/src/infoblock/components/graph/RoadAttributesGraphProvider.jsx

```diff
@@ -113,9 +116,9 @@ export default function RoadAttributesGraphProvider({ width }) {
         if (colors[label]) {
             newColor = colors[label];
         } else {
-            newColor = generatePastelColor();
+            newColor = generatePastelColor(Object.keys(colors).length);
             while (Object.values(colors).includes(newColor)) {
-                newColor = generatePastelColor();
+                newColor = generatePastelColor(Object.keys(colors).length);
             }
```

Comment:
If this while {} happens, the browser freeze, because (Object.keys(color).length) never changes here. Please recheck it and remove while {} block is it is true.